src/components/home/elements/TokenRecords.vue

```diff
@@ -136,6 +149,31 @@ export default {
             if (remove === true) {
                 this.getTokenRecords()
             }
+        },
+        async deal(result) {
```

Comment:
`deal` is meaningless. Change function name to a meaningful name, for example, `processCertifiedTokenResult`.